Userland/Services/WebContent/ConnectionFromClient.cpp

```diff
@@ -715,6 +715,7 @@ Messages::WebContentServer::TakeDocumentScreenshotResponse ConnectionFromClient:
         return { {} };
 
     auto rect = calculate_absolute_rect_of_element(page(), *document->document_element());
+    rect.set_size(m_page_host->content_size());
```

Comment:
I'd expect the document element to always be at `(0, 0)`, so perhaps we can simplify this and just turn the size into a rect at that position - might be missing something though!

Reply:
It's (20,20) on Acid3.

... Maybe this should be `set_size(content_size - doc_x, content_width - doc_y)`? Nothing bad per-se happens, but that excess get filled with the background color

Reply:
Or, maybe we should be capturing the `HTMLHtmlElement`

Reply:
> also it _is_ the document element

Ah, TIL - I though there was something else wrapping the HTML root element. LGTM then :+1: